.github/codecov.yml

```diff
@@ -23,4 +23,4 @@ comment:
   require_changes: yes		# only post when coverage changes
 
 ignore:
-  - "tests/zfs-tests"		# Don't need Tests to cover themselves
\ No newline at end of file
+  - "tests/*/**"		# Don't need Tests to cover themselves
```

Comment:
@behlendorf SHOOT, I just noticed...
In my haste to create a new PR, I butchered the syntax...

It should've been: /**/*